src/toil/batchSystems/kubernetes.py

```diff
@@ -52,8 +54,19 @@
 from toil.lib.humanize import human2bytes
 from toil.resource import Resource
 
+from toil.lib.retry import retry
+
 logger = logging.getLogger(__name__)
+     
+def exception_found(e):
```

Comment:
I would call this something like `retryable_kubernetes_errors`. There's always definitely an exception.

Also, this function needs a docstring!